docs/source/usage.rst

```diff
@@ -234,6 +244,17 @@ Disables logging during raw save. (I.e. for instance using loaddata)
 
 .. versionadded:: 2.2.0
 
+**AUDITLOG_CID_HEADER**
+
+The request header containing the Correlation ID value to use in all log entries created as a result of the request.
+
+.. versionadded:: 2.3.0
+
+**AUDITLOG_CID_GETTER**
+
+The function to use to retrieve the Correlation ID.
```

Comment:
Please mention here that the value of this setting can be a callable or a str import path

Reply:
Updated